### DISCUSSION TITLE
Does excalidraw support drawing on PDF/image?

### DISCUSSION THREAD

**Discussion by AppleFritter58** [zsviczian/obsidian-excalidraw-plugin#616] [Q&A]
It would be a very useful feature that can consolidate apps function even more.

**Comment by zsviczian** [comment]
one day maybe, but unfortunately not today. It would be great though!

**Reply by RodWarrior** [reply]
Just wanted to say this would be absolutely awesome, as it would make a tool to make linked annotations on pdf, integrating them with the whole of obsidian in a fantastic way! I'm hoping this get done, as it would be a dream come true.
